<launch>

    <node name="joy_node" pkg="joy" type="joy_node" respawn="true">
        <param name="dev" type="string" value="/dev/input/js0" />
        <param name="deadzone" value="0.06" />
				<param name="autorepeat_rate" value="50"/>
    </node>

    <node name="teleop_node" pkg="garry-robot" type="teleop_node" output="screen">
        <param name="scale_linear" value="0.35" type="double"/>
        <param name="scale_angular" value="0.09" type="double"/>
				<param name="l_offset" value="0.09"/>
				<param name="a_offset" value="0.09"/>
        <param name="timeout" value="5" type="int"/>
    </node>


</launch>
